src/realtime/methods.ts

```diff
@@ -1,29 +1,61 @@
 import { z } from "zod";
+import type { PromptAckMessage } from "./types";
 import type { WebRTCManager } from "./webrtc-manager";
 
 export const realtimeMethods = (webrtcManager: WebRTCManager) => {
 	const setPrompt = (
 		prompt: string,
-		{ enhance }: { enhance?: boolean } = {},
-	) => {
+		{ enhance, maxTimeout }: { enhance?: boolean; maxTimeout?: number } = {},
+	): Promise<boolean> => {
```

Comment:
It either resolves or rejects, resolving with a `true` boolean is redundant
```suggestion
	): Promise<void> => {
```